doc/tutorials/using_windows_as_uos.rst

```diff
@@ -144,7 +72,6 @@ Create a raw disk
 
 Run these commands on the Service VM::
 
-   # swupd bundle-add kvm-host
    # mkdir /root/img
```

Comment:
You need to install `qemu-utils` to have the `qemu-img` available on your system. Can you add this?

Reply:
add "sudo apt-get install qemu".